src/main/java/org/fcrepo/repository/WebACProtectedResources.java

```diff
@@ -0,0 +1,116 @@
+package org.fcrepo.repository;
+
+import static java.lang.Integer.MAX_VALUE;
+import static org.fcrepo.repository.FedoraDatasetImport.setAuth;
+import static org.slf4j.LoggerFactory.getLogger;
+import static javax.ws.rs.core.Response.Status.NO_CONTENT;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPatch;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.slf4j.Logger;
+
+/**
+ * Creates Fedora Resources that will be protected by the WebAC acl rules.
+ *
+ * @author whikloj
+ * @since 2015-10-01
+ */
+public class WebACProtectedResources {
+
+    private static final Logger LOGGER = getLogger(WebACProtectedResources.class);
+
+    final private String baseUrl;
+
+    final private String username;
+
+    final private String password;
+
+    final private HttpClient client;
+
+    final static private List<String> resources = Arrays.asList("webacl_box1", "box/bag/collection", "dark/archive",
```

Comment:
I was just going to dump this in favour of the ```acl_links``` map. This is unused at this point anyways.